index.html

```diff
@@ -1288,6 +1286,128 @@ <h2>DID Subject</h2>
 property. The value of <code>id</code> in the resolved <a>DID document</a> is
 expected to match the <a>DID</a> that was resolved.
       </p>
+        </section>
+
+        <section>
+        <h2>sameAs</h2>
+        <p>
+        A <a>DID controller</a> MAY assert that two or more <a>DIDs</a> identify
+        the same <a>DID subject</a> using the <code>sameAs</code> property. This
+        property is OPTIONAL.
+        </p>
+
+        <dl>
+        <dt>sameAs</dt>
+        <dd>
+The value of <code>sameAs</code> MUST be a <a
+data-cite="INFRA#lists">list</a> where each item in the list is a <a>DID</a>
```

Comment:
we might want to relax this to be a URI...

`did:example:123 owl:sameAs https://example.com/person/456`

Reply:
@OR13 Per the other discussions on this topic, I changed the property name to `sameSubject`. For reasons I will explain in a new Appendix I am composing, I favor keeping the value of `sameSubject` restricted to DIDs.